### ISSUE TITLE
Implement a Spotify-window-handler and API request to skip recorded tracks

### ISSUE THREAD

**Issue by jwallet** [jwallet/spy-spotify#154]
**What problem will this new feature resolves ?**
> Be able to interact with Spotify:
> - max out volume inside spotify
> - skip track already recorded (when detected)

**What feature or enhancement would you like to see added ?**
> Update music library quicker by recording again the same playlist that has new track

**What issue this feature could bring ?**
> When using LastFM with a window handler it would set Spotify foreground to be able to interact as a user on it, it would disturb some users to see that happening

When using:
- LastFM API
   - Use Windows handler
- Spotify API
   - Use API call to change track

---
Want to back this issue? **[Post a bounty on it!](https://issuehunt.io/r/jwallet/spy-spotify/issues/154)** We accept bounties via [IssueHunt](https://issuehunt.io/r/jwallet/spy-spotify).

**Comment by MGRed305** [comment]
the Spotify app mutes the output when I start the Spytify so my recordings are blank

**Comment by dsabella84** [comment]
@jwallet would you like some assistance on this feature, i would really benefit from this one.

**Comment by YehudaEi** [comment]
Hey,

I wrote a short script in Autoit that does this (of course this is a completely temporary solution).
I plan to soon also add it to your code ([here's an example of how to do it](https://stackoverflow.com/questions/3047375/simulating-key-press-c-sharp)).

In the meantime, whoever wants to use the code I wrote can get it from [here](https://github.com/YehudaEi/AU3-SpytifyHelper).

Notes:
No other media can be heard while the script is running.
There may be other issues, always feel free to report to me :)

Enjoy!
